scenarios/pravega-flink-autoscaling/data-generator/build.gradle

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
```

Comment:
Copyright should be Pravega authors

Reply:
I have changed this in this sample, but in the future we will need to change it in the rest of the repo.